packages/components/forms/src/base-checkbox/BaseCheckbox.tsx

```diff
@@ -90,44 +86,48 @@ function _BaseCheckbox(
     typeof isChecked !== undefined ? isChecked : defaultChecked;
 
   return (
-    <Text
-      as="label"
-      fontColor="gray900"
-      className={cx(styles.wrapper, className)}
-      htmlFor={inputId}
-      testId={testId}
-      {...otherProps}
-    >
-      <input
-        {...inputProps}
-        aria-label={ariaLabel}
-        checked={isChecked}
-        defaultChecked={defaultChecked}
-        className={styles.input}
-        type={type === 'switch' ? 'checkbox' : type}
-        onChange={onChange}
-        onFocus={handleFocus}
-        onBlur={handleBlur}
-        onKeyDown={handleKeyDown}
-        value={value}
-        disabled={isDisabled}
-        role={type}
-        aria-checked={isIndeterminate ? 'mixed' : ariaChecked}
-        ref={inputRef}
-        required={isRequired}
-        aria-required={isRequired ? 'true' : undefined}
-        aria-invalid={isInvalid ? 'true' : undefined}
-        id={inputId}
-        name={name}
-      />
-      <GhostCheckbox
-        type={type}
-        isDisabled={isDisabled}
-        isIndeterminate={isIndeterminate}
-        size={size}
-      />
-      {children}
-    </Text>
+    <div className={className}>
+      <Text
+        as="label"
+        fontColor="gray900"
+        fontWeight="fontWeightMedium"
+        className={styles.wrapper}
+        htmlFor={id}
+        testId={testId}
+        {...otherProps}
+      >
+        <input
+          {...inputProps}
+          aria-label={ariaLabel}
+          checked={isChecked}
+          defaultChecked={defaultChecked}
+          className={styles.input}
+          type={type === 'switch' ? 'checkbox' : type}
+          onChange={onChange}
+          onFocus={handleFocus}
+          onBlur={handleBlur}
+          onKeyDown={handleKeyDown}
+          value={value}
+          disabled={isDisabled}
+          role={type}
+          aria-checked={isIndeterminate ? 'mixed' : ariaChecked}
+          ref={inputRef}
+          required={isRequired}
+          aria-required={isRequired ? 'true' : undefined}
+          aria-invalid={isInvalid ? 'true' : undefined}
+          id={id}
+          name={name}
+        />
+        <GhostCheckbox
+          type={type}
+          isDisabled={isDisabled}
+          isIndeterminate={isIndeterminate}
+          size={size}
+        />
+        {children}
+      </Text>
+      {helpText && <HelpText className={styles.helpText}>{helpText}</HelpText>}
```

Comment:
a small aria improvement: I would be great if we could add `id={id + '-helptext'}` to HelpText element and add `aria-describedby={id+ '-helptext'}` to <input> when help text is defined